src/hooks/useStableSwapEstimateRemoveLiquidity.ts

```diff
@@ -68,5 +68,5 @@ export default function useStableSwapEstimateRemoveLiquidity({
     })
   }, [emptyAmounts, estimateRemoveLiquidity, estimateRemovingOneToken, withdrawTokenIndex])
 
-  return { estimatedAmounts, getEstimatedAmounts, error }
+  return { estimatedAmounts: amounts, getEstimatedAmounts, error }
```

Comment:
This has previously caused bugs due to react render cycle + async calls
```suggestion
  return { estimatedAmounts, getEstimatedAmounts, error }
```